example/demo.proto

```diff
@@ -5,7 +5,8 @@ import "google/api/annotations.proto";
 
 message Demo {
    string preprocessedField = 1 [(preprocess.field).string.trim_space = true ];
-   string untouched = 2;
+   repeated string preprocessedRepeatedField = 2 [(preprocess.field).repeated.string.trim_space = true ];
```

Comment:
Can we use same `[(preprocess.field).string.trim_space = true ];` for repeated case?

Reply:
yeah sure